test/test_proxy_tensor.py

```diff
@@ -929,6 +929,35 @@ def f(x):
         self.assertFalse(eval_guards(gm, torch.randn(25, 5)))
         self.assertExpectedInline(show_guards(gm), """L['x'].size()[0] < 20""")
 
+    def test_repeat_interleave(self):
+        def f(src_tokens, beam_size):
+            return src_tokens.repeat_interleave(beam_size, 0)
```

Comment:
Should we have a test where `beam_size` is symbolic?

Reply:
Sure I can make this symbolic too.